index.js

```diff
@@ -19,7 +22,8 @@
 // => TypeError
 // console.log("%c----------", "color: red") 
 
-
+const name = "Bob" 
+console.log(name)
```

Comment:
Good! `const` is another one of those keywords that we use to tell JavaScript that we are creating a variable. Like `let`, it's also block scoped. The special thing about `const` is that it does not allow us to re-assign the variable.